examples/reporting/parallel_report_download.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Shows how to download a set of reports from a list of accounts in parallel.
```

Comment:
suggested reword to clarify that the download is parallel, not the accounts:

"Shows how to download in parallel a set of reports from a list of accounts"